src/index.ts

```diff
@@ -66,20 +66,29 @@ module.exports = class ServerlessPlugin {
     const config = getConfig(this.serverless.service);
     if (config.enabled === false) return;
     this.serverless.cli.log("Auto instrumenting functions with Datadog");
+    validateConfiguration(config);
     setEnvConfiguration(config, this.serverless.service);
 
     const defaultRuntime = this.serverless.service.provider.runtime;
     const handlers = findHandlers(this.serverless.service, config.exclude, defaultRuntime);
+    const allLayers = { regions: { ...layers.regions, ...govLayers.regions } };
     if (config.addLayers) {
-      this.serverless.cli.log("Adding Lambda Layers to functions");
+      this.serverless.cli.log("Adding Lambda library Layers to functions");
```

Comment:
Super nit: I think `Library` should be capitalized here as well (`Lambda Library Layers`) -- technically the library is called the Datadog Lambda Library.